docs/development_suite/set-up-infrastructure/enhanced-project-workflow.md

```diff
@@ -0,0 +1,153 @@
+---
+id: enhanced-project-workflow
+title: Enhanced Project Workflow
+sidebar_label: Enhanced Project Workflow
+---
+
+:::caution
+This is a **BETA** feature and, as such, is currently under active development. Pay attention using it.
+:::
+
+The **Enhanced Project Workflow** feature aims to improve the developer experience when working with the Console, as well as pave the way for the adoption of previously unavailable opportunities.
+
+## Changes to the Console workflow
+
+The following sections will highlight the changes made to the established Console workflow, regarding in particular the new management of the project logical state, and the new deployment workflow.
+
+### New configuration management strategy
+
+The most relevant change in the Console’s behavior is that the project’s logical state will not be saved on a Git provider anymore. This will result in significant **performance improvement** during the main Console workflows, since contacting the Git provider implied a considerable overhead that will now be reduced.
```

Comment:
We have numbers from our testing environment but since the numbers for production are only theoretical we don't have something to be put here.

While I agree that statements like this should be based on numbers, in this case the architectural change itself is enough to prove the improvement is there waiting for user to migrate their projects on the new workflow